massholes/loader.py

```diff
@@ -0,0 +1,12 @@
+import pandas as pd
+
+def loader(path):
+    df = pd.read_excel(path,index_col=0)
+    treatment=pd.read_excel(path,sheet_name='y')
+    treatment=pd.Series(treatment['y'])
```

Comment:
I feel this line is redundant, simply user `treatment['y']` below. A column is already a `pd.Series`.